modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
                 It is calculated as a percentage of the overall number of event processors in the documentation and the
-                amount of event processors contained in the Mule configuration file.
-| *Coverage*    | This field represents the percentage of the event processors inside the Mule Configuration file that
-                were executed in the MUnit test.
+                amount of event processors in the Mule configuration file.
+| *Coverage*    | The percentage of the event processors inside the Mule Configuration file that is executed in the MUnit test
 |===
 
-Additionally you can click in each Resource, to get a more specific and granular report of the Event Processors executed in that resource:
+Additionally, you can click on each *Resource* to get a more specific and granular report of the Event Processors executed in that resource:
 
-image::munit-granular-coverage-report-in-anypoint-studio.png[]
+image::munit-granular-coverage-report-in-anypoint-studio.png["Granular report is highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column              | Description
-| *Name*                | The name of the container.
-| *Type*                | The type of the container.
-| *Covered Processors* | The processors inside the container that were executed during the MUnit test.
-| *Processors*         | The total amount of processors inside the container.
-| *Coverage*            | This field represents the percentage of the event processors inside the container that were executed in the MUnit test.
+| *Name*                | The name of the container
+| *Type*                | The type of the container
+| *Covered Processors* | The processors inside the container that is executed during the MUnit test
+| *Processors*         | The total amount of processors inside the container
+| *Coverage*            | The percentage of the event processors inside the container executed in the MUnit test
 |===
 
 == Disable Coverage Reports
 
-If you choose so, you can disable the coverage calculation from your project's run configuration.
+If you choose so, you can disable the coverage calculation from your project run configuration.
 
-Right click in your Package Explorer, and choose *Run As* and *Run Configurations...*, select your MUnit run configuration file to access your suite's configuration, and deselect the "Run coverage" option under your coverage settings.
+Right-click in your *Package Explorer*, and choose *Run As* and *Run Configurations...*. Select your MUnit run configuration file to access your suite configuration, and deselect the *Run coverage* option in your coverage settings.
 
-image::disable-munit-coverage-in-anypoint-studio.png[]
+image::disable-munit-coverage-in-anypoint-studio.png["MUnit run and Run coverage highlighted."]
+
+== Exclude XML Files from MUnit Coverage Report
+
+You can exclude one or more XML files from an MUnit Coverage report when testing your application. 
```

Comment:
Why would you do that?

Reply:
Should this be "MUnit coverage report" or "MUnit Coverage Report"? (i.e., is it a proper noun?)

Reply:
"MUnit Coverage Report" is a proper name and this feature is "weird".
It may be the case that developers included third-party XML files in their projects. They may need to run a report that tests their XML files so they want to exclude third-party XML files. They may want to do this because the third-party XML files do not comply with the standards, or because of any other reason you can think of (for example, they may even exclude their own XML files).

Reply:
These are good things to add to the short description (first paragraph) of the section. Give the user context as to why they should care.

Reply:
I think I'm lost. If needed, I may go over the explanation provided at https://docs.mulesoft.com/munit/latest/coverage-studio-concept, which is a different article.
Do you agree?

Reply:
I'm confused. Isn't coverage-studio-concept this topic?

All I'm saying is that we need to give context to the users on why they would do the task. Examples, use cases, reasons for doing the thing. This page gives more info on that:
https://confluence.internal.salesforce.com/x/5LmeBw

Reply:
I'm suggesting that you add some context - why would they exclude files? Something like "such as third-party XML files do not comply with standards".

Reply:
Oh! OK! Thanks!